### DISCUSSION TITLE
Infection hates null coalescing

### DISCUSSION THREAD

**Discussion by teresko** [infection/infection#1731] [Q&A] [answered]
How I am supposed to test this ?

```diff
-        return $this->list[$key] ?? null;
+        return null ?? $this->list[$key];
```

**Comment by sanmai** [comment]
Could you file a bug report?

**Reply by teresko** [reply]
Wasn't a bug, the problem was genetic.

**Comment by teresko** [comment]
Nevermind. 
I was being stupid.

I was looking a wrong set of unit tests and could not figure out why they were not covering it.

**Comment by maks-rafalko** [comment, marked as answer]
We removed this mutation in https://github.com/infection/infection/issues/1725, please upgrade.
